packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -221,6 +222,8 @@ function leaveMeeting(meetingId) {
   meeting.leave()
     .then(() => {
       meetingsCurrentDetailsElm.innerHTML = 'Not currently in a meeting';
+      // eslint-disable-next-line no-use-before-define
+      cleanUpMedia(htmlMediaElements);
```

Comment:
Am aware that I am using before it's defined to avoid restructuring code. When `meeting.leave()` is invoked the `htmlMediaElements` is already defined as it's the next code block below this one.